src/components/CampaignInitialisation/utils.ts

```diff
@@ -0,0 +1,53 @@
+type CheckVersionResult = "OK" | "OUTDATED_BINARY" | "OUTDATED_BUILD";
+type CheckVersionProps = {
+  currentBinaryVersion: string;
+  minBinaryVersion?: string;
+  currentBuildVersion: number;
+  minBuildVersion?: number;
+};
+
+const isBinaryVersionOK = (
+  currentBinaryVersion: string,
+  minBinaryVersion: string
+): boolean => {
+  const currentBinaryVersionSplit = currentBinaryVersion.split(".").map(Number);
+  const minBinaryVersionSplit = minBinaryVersion.split(".").map(Number);
+  for (let i = 0; i < minBinaryVersionSplit.length; i++) {
+    if (currentBinaryVersionSplit[i] < minBinaryVersionSplit[i]) {
+      return false;
+    } else if (currentBinaryVersionSplit[i] > minBinaryVersionSplit[i]) {
+      return true;
+    }
+  }
+  return true;
```

Comment:
hmm, so this will trigger when currentBinaryVersionSplit[i] = minBinaryVersionSplit[i]? 😆  

it's interesting to have a fallback `true` value for this function haha

Reply:
Yep when all values are the same.

What do you think will be a better approach for this?

Reply:
oooh, hmm, what if we do

```
else if (currentBinaryVersionSplit[i] >= minBinaryVersionSplit[i])
```

Reply:
Hmm that won't work for
```
currentBinaryVersion = "3.1.0"
minBinaryVersionSplit = "3.2.0"
```
since once we reach the first index (i.e. 3), it'll match an return true

Reply:
hmm that is true. hmmm.

Reply:
I cant think of anything cleaner than this HAHA.

i thought of doing 

if current index 0 value === min index 0 value, check if current index 1 value > min index 1 value
else
check and compare from current index 1 and min index 1

but i think your way is cleaner

Reply:
Why do we have to split it? Can we just `join` instead of `split` and compare the raw numbers? For example if `currentBinaryVersion` is `3.1.0` and `minBinaryVersionSplit` is `3.2.0` it'll just be comparing `310` and `320`?

Reply:
:O that may work! so converting it into a number, and then compare~

Reply:
Because it can be 3.2.123, not limited to 0-9

Reply:
ah.... so if its current is 3.2.123 and min is 3.3.0, converting to numbers will not work... ok then, let's keep it this way first HAHA

Reply:
If that is the case, `split` makes sense. Perhaps the idea of `currentBinarySplit[n] - minBinaryVersionSplit[n]` can be used so the fallback is to return `false`?

So if `currentBinarySplit[n] - minBinaryVersionSplit[n] >= 0` return `true`, else return `false`, with a fallback of return `false`?

Reply:
@yongjiajie yea that has the issue where it doesnt work for this:
```
currentBinaryVersion = "3.1.0"
minBinaryVersionSplit = "3.2.0"
expected to fail, but will pass because 3 - 3 = 0  => true
```

Reply:
I think it's best to go ahead with your approach, looks like it covers the cases it needs to, and is simple!